samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+
+
+const { TranscriptionProperties } = require('speech_to_text_api_v30');
+var SpeechToTextApiV30 = require('speech_to_text_api_v30');
+var sleep = require('system-sleep');
+var defaultClient = SpeechToTextApiV30.ApiClient.instance;
+const request = require('request');
+
+   //Create API Instance
```

Comment:
nit: no indent

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.